tests/test_migrations.py

```diff
@@ -59,6 +61,11 @@ def test_readonly_connections(self):
         migrate_app('tests', 'readonly')
         self.assertFalse(table_exists(connections['readonly'], ClickHouseTestModel))
 
+    @patch.object(RunPython, 'apply')
+    def test_hint_param(self, mock_python_exec):
+        migrate_app('tests', 'default')
+        self.assertEqual(1, mock_python_exec.call_count)
+
```

Comment:
This test does not anyhow check hints. It checks, that `RunPython` operation is called during migration. It doesn't check any of the situations:
1. Is migration applied to `default` database?
2. Is migration applied to databases other than `default`?
3. Are hints present when `Router.allow_migrate` method is called?
4. May be, this should be tested in tests for router: does router respect hints and returns default database instead of `ClickHouseModel.migrate_*_db_aliases`?

Reply:
Secondly, I would rather move this to separate `TestCase`.

Reply:
ok, I will add new tests.